src/dotnet.ts

```diff
@@ -1,32 +1,71 @@
+import * as path from 'path';
 import * as core from '@actions/core';
 import { exec } from '@actions/exec';
 import { ToolsDirectory } from './toolsDirectory';
 import { Platform } from './platform';
+import { fstat } from 'fs';
```

Comment:
Where is `fstat` being used?

Reply:
It's not, looks like it sneaked in with one of the master rebases, I'll remove it.

Reply:
removed.